src/api/integrations/managed-integrations.md

```diff
@@ -0,0 +1,123 @@
+---
+layout:  default
+grand_parent: API Reference
+parent: Integrations
+title: Managed Integrations
+permalink: /api/managed-integrations
+---
+
+# Managed Integration
+{:.no_toc}
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Managed Integration
+
+{% h4 Mandatory Fields %}
+
+|       Field        |        Type        |                                                  Description                                                   |
+| :----------------- | :----------------- | :------------------------------------------------------------------------------------------------------------- |
+| id                 | String             | The Managed Integration's unique identifier.                                                                   |
+| type               | String             | The type of Managed Integration. Supported types are `kete-enrolment`.                                         |
+| active             | Boolean            | Flag indicating whether the Managed Integration is active.                                                     |
+| state              | String             | Current state of the Managed Integration. Supported values are `pending`, `active`, `inactive` and `archived`. |
+| externalId         | String             | Field used to reference an id from an external system.                                                         |
+| claimedByAccountId | String             | Centrapay account id used to claim the Managed Integration.                                                    |
```

Comment:
Can we remove claimedByAccountId, claimedBy and claimedAt from mandatory fields, if it hasn't been claimed yet, these won't exist